bundles/org.openhab.binding.astro/src/main/java/org/openhab/binding/astro/internal/util/DateTimeUtils.java

```diff
@@ -218,20 +217,24 @@ private static Calendar adjustTime(Calendar cal, int minutes) {
      * Parses a HH:MM string and returns the minutes.
      */
     private static int getMinutesFromTime(String configTime) {
-        String time = StringUtils.trimToNull(configTime);
-        if (time != null) {
-            try {
-                if (!HHMM_PATTERN.matcher(time).matches()) {
-                    throw new NumberFormatException();
-                } else {
-                    int hour = Integer.parseInt(StringUtils.substringBefore(time, ":"));
-                    int minutes = Integer.parseInt(StringUtils.substringAfter(time, ":"));
-                    return (hour * 60) + minutes;
+        if (configTime != null) {
+            String time = configTime.trim();
+            if (!time.isEmpty()) {
+                try {
+                    if (!HHMM_PATTERN.matcher(time).matches()) {
+                        throw new NumberFormatException();
+                    } else {
+                        String[] elements = time.split(":");
+                        int hour = Integer.parseInt(elements[0]);
+                        int minutes = Integer.parseInt(elements[1]);
+                        return (hour * 60) + minutes;
+                    }
+                } catch (Exception ex) {
+                    LOGGER.warn(
+                            "Can not parse astro channel configuration '{}' to hour and minutes, use pattern hh:mm, ignoring!",
+                            time);
```

Comment:
You should make sure that you are actually catching a parse exception here, for all you know you might be catching a NPE and you wouldn't be able to tell just by looking at your logging statement.